test/api/route/messages_test.dart

```diff
@@ -304,7 +304,7 @@ void main() {
     }) async {
       connection.prepare(json: SendMessageResult(id: 42).toJson());
       final result = await sendMessage(connection,
-        destination: destination, content: content);
+        destination: destination, content: content, readBySender: true);
```

Comment:
I'd like to write this test a little differently; but probably the simplest way to express how is by just writing it. So I'll do that before merging.

(And in fact, when I go to write the style of test I'm thinking of, I discover a pre-existing behavior that's likely wrong! So I'll end up with a prep commit adding that test and a comment, then your main commit tweaked to add this new parameter to that test.)